src/Server/Middleware/TransactionMarkerMiddlewareHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @author    Craig Gosman <craig@ingenerator.com>
+ * @licence   BSD-3-Clause
+ */
+
+namespace Ingenerator\CloudTasksWrapper\Server\Middleware;
```

Comment:
Hmmm - throws up a good question re my original namespacing of all this stuff. 

This really is related to the Client side rather than the Server side - e.g. it's quite tied to the fact that the headers go under a `headers` key in the options array, as per the CreateTaskOptions. 

But equally it's also very strongly related to the middleware so I can see the logic in putting it alongside that.

50/50 really but I might lean to thinking it's more discoverable / findable in the client namespace?

Reply:
Yeah its a bit misleading